.github/workflows/ci.yml

```diff
@@ -25,8 +25,8 @@ jobs:
 
     - name: Rubocop
       run: |
-        gem install rubocop rubocop-performance --no-document
-        rubocop --require rubocop-performance --format progress
+        bundle install --without test
+        bundle exec rubocop --format progress
```

Comment:
If removed from here, should `rubocop-performance` be added to the `.rubocop.yml` file? See [doc](https://github.com/rubocop-hq/rubocop-performance#rubocop-configuration-file)

Reply:
It's already there: https://github.com/lostisland/faraday/blob/d2d52e404aa4e4d0d78a4c0a1473d3b369023b22/.rubocop.yml#L4